app/src/main/java/org/mozilla/fenix/wallpapers/WallpaperManager.kt

```diff
@@ -163,7 +139,14 @@ class WallpaperManager(
         }
     }.getOrNull()
 
-    private fun scaleBitmapToBottom(bitmap: Bitmap, view: ImageView) {
+    /**
+     * This will scale the received [Bitmap] to the size of the [view]. It retains the bitmap's
+     * original aspect ratio, but will shrink or enlarge it to fit the viewport. If bitmap does not
+     * correctly fit the aspect ratio of the view, it will be shifted to prioritize the bottom-left
+     * of the bitmap.
+     */
+    fun Bitmap.scaleBitmapToBottomOfView(view: ImageView) {
```

Comment:
nit: Should this extension instead live in a extension function file?

Reply:
This one would make more sense to move into an extension function file, IMO, because it isn't necessarily a strictly-wallpaper related function. I just always worry about polluting namespaces without an obvious need. I'd defer to your opinion if you think we should move it

Reply:
I don't feel super strongly either way, so if you feel this _will_ be used elsewhere, I'd lean that way. Otherwise, it can just live here for now.